### PR TITLE
some cleaning for the NSE table

### PR DESCRIPTION
**Pull request by zingale**
there is now a table_idx() function that can be used to get the correct index into the 1-d table.
this makes new constexpr for the table deltas
eliminates arrays to store T, rho, ye at the table points, since those are easily computed
replaces some magic numbers with more consts

### CODE REVIEW

**Comment by zingale** [comment]
this is ready for review.  It doesn't change any functionality, just makes things easier to understand / reuse

**Comment by zingale** [comment]
tests pass: http://groot.astro.sunysb.edu/Microphysics/test-suite/gfortran/2023-01-27/index.html

File: networks/aprox19/actual_network.H [hunk 1]
```diff
@@ -33,15 +33,16 @@ namespace network
...
 
-  constexpr int npts = 46221;
 
   constexpr int ntemp = 71;
   constexpr int nden = 31;
   constexpr int nye = 21;
 
-  extern AMREX_GPU_MANAGED amrex::Array1D<amrex::Real, 1, npts> ttlog;
-  extern AMREX_GPU_MANAGED amrex::Array1D<amrex::Real, 1, npts> ddlog;
-  extern AMREX_GPU_MANAGED amrex::Array1D<amrex::Real, 1, npts> yetab;
```
Comment (yut23) [inline networks/aprox19/actual_network.H:44]: These are also defined in `networks/aprox21/actual_network.H`. Should those be changed as well, or removed entirely?

File: networks/aprox19/actual_network.H [hunk 2]
```diff
@@ -33,15 +33,16 @@ namespace network
 namespace table
 {
 
-  constexpr int npts = 46221;
 
   constexpr int ntemp = 71;
   constexpr int nden = 31;
   constexpr int nye = 21;
```
Comment (aisclark91) [inline networks/aprox19/actual_network.H:40]: Moved global magic numbers into constexpr

File: sphinx_docs/source/nse.rst [hunk 3]
```diff
@@ -178,6 +178,18 @@ We determine is a zone is in NSE according to:
 
 .. _self_consistent_nse:
 
+
+NSE table ranges
```
Comment (aisclark91) [inline sphinx_docs/source/nse.rst:182]: Writing an output of the table.

**Review by aisclark91 — APPROVED** [review]
Yes, indeed, now the NSE table is clearer with this PR.

**Comment by zingale** [comment]
that namespace was removed only in `aprox21`, since the table is not meant to work with that net.

I think it is still okay in `aprox19`

**Pull request merged**